tools/custom-action/Dockerfile

```diff
@@ -7,8 +7,10 @@ RUN \
   apt update -y && \
   apt install -y locales && \
   locale-gen en_US.UTF-8 && \
-  sysctl -w kernel.unprivileged_userns_clone=1 && \
   apt install -y \
+    build-essential \
+    clang \
+    chromium \
```

Comment:
Is this needed to build the specs?
Please remove if it is not.

If you are trying to make it convenient to debug and look at generated HTML, that should happen outside the docker image.

Reply:
@dneto0 this is for Mermaid diagrams in spec which uses Puppeteer to generate the files through browser